.github/renovate.json5

```diff
@@ -1,13 +1,22 @@
 {
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   semanticCommits: "auto",
+  schedule: [
+    "after 8am on sunday"
+  ],
   enabledManagers: [
     "dockerfile",
     "gomod",
     "github-actions",
     "npm",
     "regex",
   ],
+  "packageRules": [
+    {
+      "matchManagers": ["dockerfile", "gomod", "github-actions", "npm", "regex"],
```

Comment:
I don't think we should group github-actions and regex (doesn't matter right now because there's just 1, but will matter later). The upgrades from those managers are so different from each other that they should be reviewed individually IMO.

Regarding the regex manager; I'd like to make it update the Golang version specified by GitHub Actions. I'd also like to have it update the tags specified in the `contrib/` folder

Reply:
I just wanted to introduce a quickfix for now. Until you have a more advanced setup ready.